committeeoversightapp/models.py

```diff
@@ -52,6 +53,19 @@ class Committee(models.Model):
                                      on_delete=models.CASCADE)
 
 
+class CommitteeRating(models.Model):
+    committee = models.ForeignKey(Organization, on_delete=models.CASCADE)
```

Comment:
@beamalsky right now, a committee can have multiple ratings for a single congress. is this ok? if not, i can add a unique constraint, so we don't end up with nonsensical data.

Reply:
A rating should be unique! I'm now afraid that Lugar is going to be asking for multiple types of ratings for each committee for each Congress, so it makes sense to me to hang off on that change in anticipation of having to add a "rating type" column or something similar to this model. Up to you though!

Reply:
Great instinct to hold off until we know more. I'll wait on it!